### ISSUE TITLE
ResolutionException with 1.0.0-RC11 and sbt 0.13.16

### ISSUE THREAD

**Issue by 2beaucoup** [coursier/coursier#655]
I get the following error output:
```
coursier.ResolutionException: 1 not found
    ~/.sbt/preloaded/org.scala-lang.modules/scala-parser-combinators_2.12/1.0.4/bundles/scala-parser-combinators_2.12.jar
coursier.ResolutionException: 1 not found
    ~/.sbt/preloaded/org.scala-lang.modules/scala-parser-combinators_2.12/1.0.4/bundles/scala-parser-combinators_2.12.jar
[error] (nested-project1/*:update) coursier.ResolutionException: 1 not found
[error]    ~/.sbt/preloaded/org.scala-lang.modules/scala-parser-combinators_2.12/1.0.4/bundles/scala-parser-combinators_2.12.jar
[error] (nested-project2/*:update) coursier.ResolutionException: 1 not found
[error]     ~/.sbt/preloaded/org.scala-lang.modules/scala-parser-combinators_2.12/1.0.4/bundles/scala-parser-combinators_2.12.jar
```
Also reported here:
- https://github.com/circe/circe/issues/736#issuecomment-325487517
- https://github.com/sbt/sbt/issues/3412#issuecomment-321025292

Anything I can do to help analyzing the issue?

**Comment by alexarchambault** [comment]
https://github.com/coursier/coursier/commit/ef669e68e493f0fd9044926ed717e32ef4f2196a should solve that.

As a workaround for now, you can just wipe your `~/.sbt/preloaded/org.scala-lang.modules/scala-parser-combinators_2.12/1.0.4/`. It's only some kind of cache (~preloaded one), set up by sbt, and that happens to confuse coursier sometimes.

**Comment by alexarchambault** [comment]
Fixed in `1.0.0-RC12`. Re-open if that's not the case for you.
